### DISCUSSION TITLE
Install LSUClient on a remote computer and run get-lsupdate | install-lsupdate -passthrough

### DISCUSSION THREAD

**Discussion by Ravagon** [jantari/LSUClient#100] [Ideas]
Install LSUClient on a remote computer and run get-lsupdate | install-lsupdate -passthrough
If this is possible?

**Comment by jantari** [comment]
That's definitely possible, you just run `Install-Module -Name 'LSUClient'` and then whatever else you want to run on the remote computer.

To run commands on remote computers you use whatever tool you like (or already have) - InTune, ssh, WinRM, psexec, PDQ Deploy, GPO startup script, your RMM, your EDR, your MDM - any method at all you can think of that allows you to run remote PowerShell commands. There's way too many to list them all.

**Reply by Ravagon** [reply]
I was thinking more like this after installing the module on the remote computer i just run get-lsupdate | install-lsupdate -computername computername -credentials user password so I see which drivers and if BIOS is updated and required a reboot. 
As it is now i have to save the output to a file locally on the remote computer then collect the output from that instead of getting the output directly on my powershell console. Or if you can somehow pipe the output from install-lsupdate back to my powershell console from the remote computer somehow.

**Reply by jantari** [reply]
You can just do:

```powershell
Invoke-Command -ComputerName computer -ScriptBlock {
    Get-LSUpdate -Verbose
} -Credential user
```

This does what you want, and works for any commands.

But, running independently on the remote machine and saving the output to a file like you're doing now has a big advantage. You aren't dependent on your network connection to stay alive. If the remote machine installs a network driver, you'll probably lose connection for a bit which means you're missing output. If you log to a file and check it later you won't miss anything. Just something to consider.

**Comment by Ravagon** [comment]
That would work somewhat I have to save each computer in a separate log file for each computer on my computer and join em together  when all are done. Well Powershell is pretty resilient if it drops connection it will try reconnect for 4mins before giving up and network drivers installs pretty fast and the computer comes back online and the script continues unless the infamous user turns the computer off.
I'm pretty new at this Powershell scripting but it feels darn powerful.
Would be nice if the command can take a text file as input and run on multiple computers at the same time.
Thanks for the help! :)
